.github/workflows/test-release-wf.yml

```diff
@@ -0,0 +1,98 @@
+name: Test-Release (DELETE LATER)
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to tag the release with, e.g., 1.2.0, 1.3.0
+        required: true
+
+permissions:
+  id-token: write
+  contents: write
+
+jobs:
+    test-build-pack-release:
+        runs-on: windows-latest
+
+        env:
+            DOTNET_MULTILEVEL_LOOKUP: 1
+
+        strategy:
+            fail-fast: true
+
+        steps:
+        - name: Checkout repository
+          uses: actions/checkout@v2
+
+        - name: Assume role to get .snk file
+          uses: aws-actions/configure-aws-credentials@v1
+          with:
+            role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_ARN_RELEASE }}
+            aws-region: us-west-2
+
+        - name: Get Strong-Naming Key
+          run: >
+            aws s3api get-object
+            --bucket ${{ secrets.SNK_BUCKET }}
+            --key awsxrayrecorder.dll.snk
+            awsxrayrecorder.dll.snk
+          
+        - name: Install dependencies
+          run: dotnet restore .\sdk\AWSXRayRecorder.sln
+
+        - name: Build solution
+          run: >
+            dotnet build
+            .\sdk\AWSXRayRecorder.sln
+            /p:Configuration=Release
+            --no-restore
+            /p:AssemblyOriginatorKeyFile=${{ github.workspace }}\awsxrayrecorder.dll.snk
+
+        - name: Clean up the snk file
+          run: rm .\awsxrayrecorder.dll.snk
+
+        - name: Assume signer role
+          uses: aws-actions/configure-aws-credentials@v1
+          with:
+            role-to-assume: ${{ secrets.AWS_ARTIFACT_ACCESS_ROLE_ARN }}
+            aws-region: us-west-2
+
+        - name: Invoke Signing script
+          env:
+            UNSIGNED_BUCKET: ${{ secrets.AWS_UNSIGNED_BUCKET_NAME }}
+            SIGNED_BUCKET: ${{ secrets.AWS_SIGNED_BUCKET_NAME }}
+          #TODO: There is probably a better way to pass in a list of paths as a single parameter to the script.
+          run: |
+            .\buildtools\sign_files.ps1 -Filters AWSXRayRecorder.*.dll -Recurse -Path .\sdk\src\Core\bin\Release
+            .\buildtools\sign_files.ps1 -Filters AWSXRayRecorder.*.dll -Recurse -Path .\sdk\src\Handlers\AspNet\bin\Release
+            .\buildtools\sign_files.ps1 -Filters AWSXRayRecorder.*.dll -Recurse -Path .\sdk\src\Handlers\AspNetCore\bin\Release
+            .\buildtools\sign_files.ps1 -Filters AWSXRayRecorder.*.dll -Recurse -Path .\sdk\src\Handlers\AwsSdk\bin\Release
+            .\buildtools\sign_files.ps1 -Filters AWSXRayRecorder.*.dll -Recurse -Path .\sdk\src\Handlers\EntityFramework\bin\Release
+            .\buildtools\sign_files.ps1 -Filters AWSXRayRecorder.*.dll -Recurse -Path .\sdk\src\Handlers\SqlServer\bin\Release
+            .\buildtools\sign_files.ps1 -Filters AWSXRayRecorder.*.dll -Recurse -Path .\sdk\src\Handlers\System.Net\bin\Release
+            
+        - name: Pack nugets
```

Comment:
Nah. not really needed but I kept them anyway in case I need to look at the packaged nugets for signatures. I don't think there's any harm in keeping these steps.